packages/ui/src/components/w3m-button/styles.ts

```diff
@@ -1,29 +1,11 @@
-import { css, html } from 'lit'
+import { html } from 'lit'
+import { scss } from '../../style/utils'
 import { color } from '../../utils/Theme'
+import scssStyles from './styles.scss'
 
 // -- static styles ------------------------------------------------ //
-export default css`
-  .w3m-button {
-    border-radius: 28px;
-    height: 28px;
-    padding: 0 10px;
-  }
-
-  .w3m-button::after {
-    border-radius: inherit;
-  }
-
-  .w3m-button:disabled::after {
-    background-color: transparent;
-  }
-
-  .w3m-button-icon-left svg {
-    margin-right: 5px;
-  }
-
-  .w3m-button-icon-right svg {
-    margin-left: 5px;
-  }
+export default scss`
+  ${scssStyles}
```

Comment:
Not a huge fan of this, it seems to be adding complexity rather than moving it away. Why do we need to do this again?

I was under assumption that we could just do following
```
import styles from "./styles.scss"

// Inside LitElement class
styles = style
```

And for now only leave `dynamicStyles` in the .ts file

Reply:
We could actually do, with the current structure

```ts

import styles from './styles.scss'

// inside LitElement class
styles = scss`${style}` 
// or
styles = scss(style)
```

I just kept it in `styles.ts` because it made sense to me to have one place where we import static and dynamic styles. In the future we could just have this in the class